drivers/serial/uart_stm32.c

```diff
@@ -137,15 +146,13 @@ static inline void uart_stm32_set_baudrate(const struct device *dev, uint32_t ba
 
 	/* Get clock rate */
 	if (IS_ENABLED(STM32_UART_DOMAIN_CLOCK_SUPPORT) && (config->pclk_len > 1)) {
-		if (clock_control_get_rate(data->clock,
-					   (clock_control_subsys_t)&config->pclken[1],
+		if (clock_control_get_rate(data->clock, (clock_control_subsys_t)&config->pclken[1],
 					   &clock_rate) < 0) {
 			LOG_ERR("Failed call clock_control_get_rate(pclken[1])");
 			return;
 		}
 	} else {
-		if (clock_control_get_rate(data->clock,
-					   (clock_control_subsys_t)&config->pclken[0],
+		if (clock_control_get_rate(data->clock, (clock_control_subsys_t)&config->pclken[0],
```

Comment:
Please don't mix format changes and functional changes.
Make a dedicated commit if for format changes

Reply:
And I'd actually suggest to drop the format changes as the don't add anything.
Note that if you rebase on top of main, you'll see that they are not suggested anymore